transifex/native/parsing.py

```diff
@@ -36,7 +36,9 @@ def __init__(self, string, _context=None, **meta):
         :param unicode _context: an optional context that accompanies
             the source string
         """
-        self.key = generate_key(string, context=_context)
+
+        _, plurals = parse_plurals(string)
+        self.key = generate_key(plurals, context=_context)
```

Comment:
Isn't `generate_key`'s signature expecting a string as the first argument and plurals as an optional kwarg? Relevant to a previous comment.

Reply:
Wow. Dunno how I missed that. Curious why this works as is.